x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/privilege_form_calculator/privilege_form_calculator.ts

```diff
@@ -95,18 +94,25 @@ export class PrivilegeFormCalculator {
    * @param featureId the feature id
    * @param privilegeIndex the index of the kibana privileges role component
```

Comment:
nit: missing jsdocs for new `allSpacesSelected` parameter

```suggestion
   * @param privilegeIndex the index of the kibana privileges role component
   * @param allSpacesSelected indicates if the privilege form is configured to grant access to all spaces.
```